misc/sparse_matrix.py

```diff
@@ -0,0 +1,60 @@
+import taichi as ti
+
+ti.init(arch=ti.x64)
+
+n = 8
+
+K = ti.SparseMatrixBuilder(n, n, max_num_triplets=100)
+f = ti.SparseMatrixBuilder(n, 1, max_num_triplets=100)
+
+
+@ti.kernel
+def fill(A: ti.SparseMatrixBuilder, b: ti.SparseMatrixBuilder,
```

Comment:
@squarefk That's a good point. I think we need to unify these type hint conventions sometime. Not sure if using `ti.sparse_matrix_builder()` is optimal since that introduces one more token to `ti`. Either way is fine for now though.